crates/kubelet/src/config.rs

```diff
@@ -633,6 +657,9 @@ mod test {
         assert_eq!(config.allow_local_modules, true);
         assert_eq!(config.node_labels.len(), 2);
         assert_eq!(config.node_labels.get("label1"), Some(&("val1".to_owned())));
+        assert_eq!(config.insecure_registries.clone().unwrap().len(), 2);
+        assert_eq!(&config.insecure_registries.clone().unwrap()[0], "local");
```

Comment:
Nit: Because this is the end of the test, you could just do something like this to make things a bit less gnarly:

```rust
let insecure_registries = config.insecure_registries.expect("registries")
...do assertions here
```

Reply:
Might skip that because it could become a source of unrelated changes if a subsequent PR added test lines after these ones... this whole area could do with a bit more granularity though, to avoid this sort of gnarl!